modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -298,7 +301,9 @@ public String getSecretKey() {
      * @return a default region
      */
     public String getRegion() {
-        return "us-east-1";
+        return this.getEnvMap().containsKey("DEFAULT_REGION")?
```

Comment:
There is Map.`getOrDefault()` I think, would it work?

Reply:
I'll try to change it, today

Reply:
Done